scripts/utils/formatElmCompilerErrors.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var chalk = require('chalk');
+
+function stripRedundantInfo(error) {
+  return error.replace(
+    /Module build failed: Error: Compiler process exited with error Compilation failed/g,
+    ''
+  );
+}
+
+module.exports = function formatElmCompilerErrors(messages) {
+  var errors = messages.errors;
+  var warnings = messages.warnings;
```

Comment:
Why not to use `const` and destructuring?

Reply:
@fobos this one is supposed to be running in the browser and under the node(without being transpiled by Babel)